client/tx/search.go

```diff
@@ -172,15 +176,15 @@ func SearchTxRequestHandlerFn(cliCtx context.CLIContext, cdc *codec.Codec) http.
 	return func(w http.ResponseWriter, r *http.Request) {
 		var tags []string
 		var page, limit int
-		var txs []sdk.TxResponse
+		var pageableResult *sdk.SearchTxsResult
```

Comment:
Why are we returning a pointer to `SearchTxsResult`?

Reply:
Because SearchTxs() return a pointer. :)